src/reconstruction/modelGeneration/fastLeakTest.m

```diff
@@ -53,18 +50,24 @@
     rxnNames = '';
 end
 modelexchangesAbbr = unique([modelexchangesAbbr;rxnNames']);
-TestRxnNum = length(modelexchangesAbbr)
+TestRxnNum = length(modelexchangesAbbr);
 FluxExV =[];
 while cnt == 1
     modelClosed = changeObjective(modelClosed,modelexchangesAbbr);
     FF2=optimizeCbModel(modelClosed,'max');
-   ObjValue = FF2.f
+    if FF2.stat == 0
```

Comment:
Shouldn't this read: `if FF2.stat ~= 1` ?

Reply:
No. The Error thrown here is only for an infeasible model. 
Admittedly, for other non optimal solutions there should probably also be an error, but they would mean very different things.

Reply:
The value of `FF2.f` when `FF2.stat == -1` or `FF2.stat == 2`  is `0`. Is that intended behaviour?

Reply:
I'd say no. I added a error message for those instances.

Reply:
Thanks!